sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_models.py

```diff
@@ -201,3 +224,114 @@ def _from_generated(cls, result_info):
             subcode=result_info.subcode,
             message=result_info.message,
         )
+
+class CreateCallResult(msrest.serialization.Model):
+    """The response payload of the create call operation.
+
+    :param call_connection_id: The call connection id.
+    :type call_connection_id: str
+    """
+
+    def __init__(
+        self,
+        **kwargs # type: Any
+    ): # type: (...) -> None
+        self.call_connection_id = kwargs['call_connection_id']
+    
+    @classmethod
+    def _from_generated(cls, create_call_result):
+        if create_call_result is None:
+            return None
+
+        return cls(
+            call_connection_id = create_call_result.call_connection_id
+        )
+
+class JoinCallResult(msrest.serialization.Model):
+    """The response payload of the join call operation.
+
+    :param call_connection_id: The call connection id.
+    :type call_connection_id: str
+    """
+
+    def __init__(
+        self,
+        **kwargs # type: Any
+    ):
+        self.call_connection_id = kwargs['call_connection_id']
+
+    @classmethod
+    def _from_generated(cls, join_call_result):
+        if join_call_result is None:
+            return None
+
+        return cls(
+            call_connection_id = join_call_result.call_connection_id
+        )
+
+class StartCallRecordingResult(msrest.serialization.Model):
+    """The response payload of start call recording operation.
+
+    :param recording_id: The recording id of the started recording.
+    :type recording_id: str
+    """
+
+    def __init__(
+        self,
+        **kwargs # type: Any
+    ):
+        self.recording_id = kwargs['recording_id']
+
+    @classmethod
+    def _from_generated(cls, start_call_recording_result):
+        if start_call_recording_result is None:
+            return None
+
+        return cls(
+            recording_id = start_call_recording_result.recording_id
+        )
+
+class CallRecordingProperties(msrest.serialization.Model):
+    """The response payload of get call recording properties operation.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :param recording_state: Required. The state of the recording. Possible values include:
+     "active", "inactive".
+    :type recording_state: str or ~azure.communication.callingserver.models.CallRecordingState
+    """
+
+    _validation = {
+        'recording_state': {'required': True},
+    }
+
+    _attribute_map = {
+        'recording_state': {'key': 'recordingState', 'type': 'str'},
+    }
+
+    def __init__(
+        self,
+        **kwargs # type: Any
+    ):
```

Comment:
I have concerns about creating these object model.  
a good reference can be taken from JAVA SDK.
msrest.serialization.Model seems providing methods for internal usage only, I don't think we need  inherit this class.

Reply:
Made changes and removed inheriting from msrest.serialization.Model